spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/core/AppDeploymentRequest.java

```diff
@@ -34,11 +34,12 @@
  * app is executed, such as allowed memory, cpu or simply a way to define
  * colocation like node labeling.
  *
- * For passing properties or parameters into the app itself, use
- * {@link AppDefinition#getProperties()}.
+ * For passing properties into the app itself, use {@link AppDefinition#getProperties()}.
+ * Those will typically be passed as env vars, unless an alternative approach is better
+ * suited to the target platform. Each deployer implementation should clearly document
+ * how it handles these properties.
  *
```

Comment:
I would say they could be passed as env vars or as --key=value style command line args